main_comment.go

```diff
@@ -0,0 +1,94 @@
+package main
+
+import (
+	"strconv"
+	"strings"
+
+	"github.com/davecgh/go-spew/spew"
+	"github.com/gin-gonic/gin"
+	"github.com/google/go-github/v28/github"
+
+	clientgithub "github.com/mendersoftware/integration-test-runner/client/github"
+)
+
+func processGitHubComment(ctx *gin.Context, comment *github.IssueCommentEvent, githubClient clientgithub.Client, conf *config) error {
+	log := getCustomLoggerFromContext(ctx)
+
+	// process created actions only, ignore the others
+	action := comment.GetAction()
+	if action != "created" {
+		log.Infof("Ignoring action %s on comment", action)
+		return nil
+	}
+
+	// accept commands only from organization members
+	if member := githubClient.IsOrganizationMember(ctx, githubOrganization, comment.Sender.GetLogin()); !member {
```

Comment:
Nit:
`member is stored as a bool and never used`

Reply:
fixed